docs/guides/pagination.md

```diff
@@ -4,4 +4,124 @@ title: Pagination
 
 # Pagination
 
-Documentation coming soon
+Pagination is a common use case in APIs to efficiently return some results of a relationship instead of all of them, since that may be unperformant.
+
+GraphQL as a spec [recommends cursor based pagination](https://graphql.org/learn/pagination/) and refers to [Relay's Connection Spec](https://relay.dev/graphql/connections.htm) for specific implementation details.
+
+Here we show a minimal example of how you can leverage Strawberry's generic Types
+to build the types required to comply with the relay spec.
+
+```python
+
+import base64
+from typing import List, Generic, TypeVar, Optional
+
+import strawberry
+from strawberry.arguments import UNSET
+
+
+GenericType = TypeVar("T")  # type: ignore warning as 'T' is too succint to be meaningful for humans
```

Comment:
I think using `T` is common enough this isn't an issue

Reply:
Let me disagree on this one, it's common when you have dealed with generic types and type hinting, but
for a tutorial aimed for begginers, i prefer to avoid acronyms and be specific on what this line does